### PR TITLE
8319948: jcmd man page needs to be updated

### PR DESCRIPTION
**Pull request by dholmes-ora**
Please review these missing updates to the `jcmd` manpage - see JBS issue for details.

I also fixed the sub-command ordering in a few places and a couple of minor formatting fixes.

Thanks to @tstuefe for collating the initial information on what was missing. I used to `jcmd help <cmd>` option to compare the help text with the manpage text and adjusted the manpage accordingly (and found a few more omissions - some of which will be fixed by follow up RFE by the respective owners).

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8319948](https://bugs.openjdk.org/browse/JDK-8319948): jcmd man page needs to be updated (**Bug** - P4)

### Reviewers
 * [Kevin Walls](https://openjdk.org/census#kevinw) (@kevinjwalls - Committer) ⚠️ Review applies to [9e47816e](https://git.openjdk.org/jdk/pull/17213/files/9e47816eeedb9d50b6ebefa99316fd1d5432dfda)
 * [Alan Bateman](https://openjdk.org/census#alanb) (@AlanBateman - **Reviewer**)

### Contributors
 * Thomas Stuefe `<stuefe@openjdk.org>`

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17213/head:pull/17213` \
`$ git checkout pull/17213`

Update a local copy of the PR: \
`$ git checkout pull/17213` \
`$ git pull https://git.openjdk.org/jdk.git pull/17213/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17213`

View PR using the GUI difftool: \
`$ git pr show -t 17213`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17213.diff">https://git.openjdk.org/jdk/pull/17213.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17213#issuecomment-1873644644)

### CODE REVIEW

**Review by kevinjwalls — APPROVED** [review]
Looks good.

Trivially can note that here in jcmd.1 VM.flags says "Prints the VM flag..." while diagnosticCommand.hpp says: "Print VM flag...".  
We could make it consistent, but not important.
"Print" maybe in the minority compared to "Prints", but we have some of each. 8-)

**Comment by dholmes-ora** [comment]
Thanks for the review @kevinjwalls . I did a consistency sweep over the VM subcommands and changed a lot of `Prints` to `Print` so the manpage matches the help text. Which form is better grammatically I'm not sure, but if we want to change things we should do that for the help text and manpage at the same time.

File: src/jdk.jcmd/share/man/jcmd.1 [hunk 1]
```diff
@@ -178,58 +195,71 @@ Prints all compiled methods in code cache that are alive.
 Impact: Medium
 .RE
 .TP
-\f[V]Compiler.perfmap\f[R] [\f[I]arguments\f[R]] (Linux only)
-Write map file for Linux perf tool.
+\f[V]Compiler.directives_add\f[R] \f[I]filename\f[R] \f[I]arguments\f[R]
+Adds compiler directives from a file.
```
Comment (AlanBateman) [inline src/jdk.jcmd/share/man/jcmd.1:199]: I might be misreading this one, it looks like there are two arguments but the the help output shows just one, maybe check that one.

Comment (AlanBateman) [reply src/jdk.jcmd/share/man/jcmd.1:199]: I didn't spot anything else, the update looks good to me.

**Review by AlanBateman — APPROVED** [review]

**Comment by dholmes-ora** [comment]
Thanks for the review @AlanBateman !

**Pull request closed without merging**